src/ZfBugsnag/Service/BugsnagService.php

```diff
@@ -67,33 +67,49 @@ public function setupErrorHandlers()
      */
     public function errorHandler($errno, $errstr, $errfile = '', $errline = 0, $errcontext = [])
     {
-        $result = $this->bugsnag->errorHandler($errno, $errstr, $errfile, $errline);
+        try
+        {
+            $this->bugsnag->errorHandler($errno, $errstr, $errfile, $errline);
+        } catch (\Exception $e) {
+            // Something wrong in the bugsnag notify call
```

Comment:
@nickurt not yet, I did spend some time looking around at other open source libraries to see what they might do in this situation, particular ZF2 modules, but I haven't found anything concrete yet.  I'll put together a response of what I'm able to find soon.